### ISSUE TITLE
STA/OPR 9750 katariasiddhi MiniProject #02

### ISSUE THREAD

**Issue by katariasiddhi** [katariasiddhi/STA9750-2024-FALL#3]
Hi @michaelweylandt!

I've uploaded my work for MiniProject #02 - check it out!

https://katariasiddhi.github.io/STA9750-2024-FALL/mp02.html

**Comment by michaelweylandt** [comment]
Thanks @katariasiddhi !

Peer Feedback:

- @fatima8808
- @Victor-Louie
- @Malexandra-Jerez
- @chricannon

It is now time for the peer feedback round for Mini-Project 02. Please
review @katariasiddhi's submission for this mini-project and provide peer feedback.

Using the rubric at https://michael-weylandt.com/STA9750/miniprojects/mini02.html#rubric,
please grade their submission out of a total of 50 points. Note that this
rubric is slightly different than that used for Mini-Project 01.

For each of the five categories, please give them a separate score and provide
a total (sum) score across the entire assignment. Feel free to assign extra
credit if you feel it is warranted (following the rubric).

If you give a score of less than 5 for any category, please provide a suggestion
for improvement. (You can also give suggestions for any element they did well -
more feedback is always great!)

As you go through this peer feedback exercise, think about what you particularly
like about this submission and how you can incorporate that approach in your future
work. If something is particularly insightful or creative, give some kudos!

Evaluators: This should take you around 15 minutes per peer feedback. You are
not required to engage in substantial back-and-forth with @katariasiddhi, but you are of course
welcome to initiate a discussion.

@katariasiddhi: please engage fully with your peers. They are here to help you!

Submission URL should be: https://katariasiddhi.github.io/STA9750-2024-FALL/mp02.html

Feel free to link to other repos, the course documentation, or other useful
examples.

Thanks!
@michaelweylandt

CC: @charles-ramirez

**Comment by Malexandra-Jerez** [comment]
hi @katariasiddhi <https://github.com/katariasiddhi> !

Please see below review.

Written communication:  8, great intro and pitch. Mostly code. You did
include titles. It looks like from the code you generated the answers.
Maybe a breakdown or separate answers from code on plain text would be
nice. It gets me confused and gives me a sense of overwhelm [maybe b/c not
used to code as much yet]. ; separate answer on text format.

Project Skeleton: 8, May have missed some tasks. Perhaps you performed the
formatting of columns, but I didn't see the code for it. Again, a lot of
code and titles makes it a little too busy and hard to identify something
specific.

Formatting and Display: 6, Ton of code, one colorful chart. Perhaps adding
more graphs or tables.

Code Quality: 9,  short and sweet, to the point.

Data preparation: 10

total = 41


Kindly,
Alex

**Comment by Victor-Louie** [comment]
hi @katariasiddhi 

Please see below review.

Written communication:  5, The report was mainly written in code. The other written communication was in the introduction and pitch. I can see that the code generated the answers but, I would give an explanation/breakdown whenever necessary after your code. It was also extremely hard to follow because there were error messages on the report when there should not be. 

Project Skeleton: 5, I believe you are missing the code for the entire genre section and validation for the success score metric?

Formatting and Display: 5, There is only bar chart. I believe we were supposed to have 2 as a minimum. Adding more graphs or tables would help the report be more clear.

Code Quality: 10,  short and concise.

Data preparation: 10

total = 35

**Comment by fatima8808** [comment]
Hi @katariasiddhi,

Nice job in completing your mini project 2. Overall, I think there are some areas of improvement in regards to building graphical visualizations and written communication. You began your project with a nice introduction explaining the purpose of the project, and it would’ve been ideal to continue your commentary and explanations throughout the project. Please see a detailed breakdown below and suggestions for improvement that may be helpful.

**Written Communication: 6**
Like others have mentioned and as I explained briefly earlier, there wasn’t too much of an explanation of your findings when answering the task-specific questions. I think it would be super helpful to the reader if you included more insight instead of inputting and running your code. Your heading titles did help with creating some form of organization. However, with raw code and output, it makes it difficult to understand the context or motivation for those unfamiliar with the project instructions.

**Project Skeleton: 6**
You answered most tasks provided in the project instructions, however you didn’t include any code in validating your success metric which is an important task before deciding which actors/directors you would like to choose to remake a classic movie. I think you also missed the question regarding Mark Hamill’s top four projects, and it looks like some parts of your code is incorrect or incomplete (finding the average ratings for Happy Days shows the ratings only for seasons 1-8 and there are 11 seasons). I think you’re on the right track, but there’s some improvement to be made in this area. 

**Formatting & Display: 6**
You included one graphical visualization which is clear to understand and easy to read, however I think it can be made better by filtering for average ratings of movies that these actors/actresses/directors are well-known because it looks like you created a graph based on financial data (box office amounts) when our data is non-financial. I think another way you can improve your display is to use code folds to hide long lines of code, “tibbles” and outputs that should be substituted with written communication. I would also suggest looking into the “gt” package to create nice tables since there weren’t any tables made.

**Code Quality: 9**
Your code was easy to understand with well-chosen variable names and commentary for each question being answered. 

**Data Preparation:** 10. Automatic (10/10). Out of scope for this mini-project

Total: 37

Like every and any project we complete, there'll always room for improvement so I hope this feedback will be helpful in the next mini project, keep going!

**Comment by chricannon** [comment]
Hi @katariasiddhi! Good job on MP #02! I enjoyed reviewing your project. 

Written Communication: 6/10
You included an intro and written communication for tasks 6 and 7, but there was not any commentary throughout the rest of your project. 
Including commentary within each section/task and questions (if applicable) would improve your project and allow for a reader to better understand each block of code and section. While the code and output answers many of the questions, the key steps and findings are not explained. 

Project Skeleton: 6/10
Missing code for validating success metric, which is a skipped task. Due to this skipped task, I had to deduct some points unfortunately.

Formatting & Display: 6/10
The formatting of your code could be elevated using code folding in some sections. 

Also, using the gt package for some of your outputs, to put your code output in table format, would elevate your formatting and display. 

I like your one bar chart and use of a color schema for it. A few more visualizations would have helped with the display and readability of your project. 

Code Quality: 10/10
Code is not overly complicated, easy to read and understand. Good job! 

Data Preparation: 10/10 (automatic)


**Total: 38/50**
